core/rend/vulkan/pipeline.h

```diff
@@ -86,38 +86,36 @@ class DescriptorSets
 			perPolyDescSet = perPolyAlloc.alloc();
 			std::vector<vk::WriteDescriptorSet> writeDescriptorSets;
 
-			vk::DescriptorImageInfo imageInfo;
 			if (poly.texture != nullptr)
 			{
-				imageInfo = vk::DescriptorImageInfo(samplerManager->GetSampler(poly.tsp),
+				vk::DescriptorImageInfo imageInfo(samplerManager->GetSampler(poly.tsp),
 						((Texture *)poly.texture)->GetReadOnlyImageView(), vk::ImageLayout::eShaderReadOnlyOptimal);
-				writeDescriptorSets.emplace_back(perPolyDescSet, 0, 0, 1, vk::DescriptorType::eCombinedImageSampler, &imageInfo, nullptr, nullptr);
+				writeDescriptorSets.emplace_back(perPolyDescSet, 0, 0, vk::DescriptorType::eCombinedImageSampler, imageInfo);
```

Comment:
imageInfo is used by reference and goes out of scope

Reply:
Fixed